src/structures/Message.js

```diff
@@ -168,8 +168,8 @@ class Message extends Base {
             inviteCodeExp: data.inviteCodeExp,
             groupId: data.inviteGrp,
             groupName: data.inviteGrpName,
-            fromId: data.from._serialized,
-            toId: data.to._serialized
+            fromId: data.from._serialized ? data.from._serialized : data.from,
```

Comment:
You will get a TypeError in a case of `data.from` is null or undefined or if `data.from` is a property but not an object

Reply:
Yes, I know that, it's just that I'm trying to make the code not too long and simpler